source/development/makefile.rst

```diff
@@ -86,157 +86,163 @@ There are other targets used to get information about the Makefile, but they won
+|                              | Default value    | wazuh                                                                                               |
+|                              +------------------+-----------------------------------------------------------------------------------------------------+
+|                              | Allowed values   | Any string.                                                                                         |
++------------------------------+------------------+-----------------------------------------------------------------------------------------------------+
+| **DISABLE_SYSC**             | Disable the compilation of the Syscollector module.                                                                    |
+|                              +------------------+-----------------------------------------------------------------------------------------------------+
+|                              | Default value    | n/a                                                                                                 |
+|                              +------------------+-----------------------------------------------------------------------------------------------------+
+|                              | Allowed values   | 1, yes, YES, y, Y                                                                                   |
++------------------------------+------------------+-----------------------------------------------------------------------------------------------------+
+| **DISABLE_CISCAT**           | Disable the compilation of the CIS-CAT module.                                                                         |
+|                              +------------------+-----------------------------------------------------------------------------------------------------+
+|                              | Default value    | n/a                                                                                                 |
+|                              +------------------+-----------------------------------------------------------------------------------------------------+
+|                              | Allowed values   | 1, yes, YES, y, Y                                                                                   |
++------------------------------+------------------+-----------------------------------------------------------------------------------------------------+
+| **DISABLE_STRIP_SYMBOLS**    | Disable debug symbols stripping from binaries. Supported operating systems: GNU/Linux, Windows and macOS.              |
+|                              +------------------+-----------------------------------------------------------------------------------------------------+
+|                              | Default value    | n/a                                                                                                 |
```

Comment:
shouldn't this be "no"?

Reply:
Unfortunately no! The documentation does not use default values ​​here for all fields!